traffic_monitor/cache/cache.go

```diff
@@ -154,6 +154,9 @@ func ComputedStats() map[string]StatComputeFunc {
 		"kbps": func(info ResultInfo, serverInfo tc.TrafficServer, serverProfile tc.TMProfile, combinedState tc.IsAvailable) interface{} {
 			return info.Vitals.KbpsOut
 		},
+		"gbps": func(info ResultInfo, serverInfo tc.TrafficServer, serverProfile tc.TMProfile, combinedState tc.IsAvailable) interface{} {
+			return info.Vitals.KbpsOut / 1000000
```

Comment:
are we worried about losing precision here? ex ~ 1500000 kbps would be represented as 1 gbps

Reply:
I agree, gbps is too big, I think we need a float64 here. I think the Monitor will properly cast any numeric primitive to a float, but it'll need tested to be sure.

Reply:
I'm gonna cast it as float64 just to be safe.